io.github.syco.blink.json

```diff
@@ -0,0 +1,384 @@
+{
+  "id": "io.github.syco.blink",
+  "command": "/app/bin/blink",
+  "base": "io.qt.qtwebkit.BaseApp",
+  "base-version": "5.11",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
```

Comment:
Update runtime to 5.14

Reply:
Yeah, for sure this should either be an LTS runtime or the very latest.